src/scenes/Home/Home.tsx

```diff
@@ -56,7 +58,8 @@ const StyledPage = styled.div<{ navbarHeight: number }>`
   }
 
   @media only screen and ${breakpoints.desktop} {
-    max-width: 1200px;
+    // max-width: 1200px;
+    max-width: 1336px;
```

Comment:
todo: this is to accommodate the new Featured Page's intended width. double check how this impacts feed once feed is working on dev again @kaitoo1